### PR TITLE
Create French Lingala Baseline

### PR DESCRIPTION
**Pull request by espoirMur**
# French to Lingala

Author: Murhabazi B. Espoir

## Data

	- The JW300 French - Lingala.

## Model

	- Default Masakhane Transformer translation model.
	- The model can be found <a href="http://34.66.168.159/french_lingala_espoir/models/frln_transformer/340000.ckpt">Here</a>

## Analysis

Example 1
```ln
   	Source:     Peut - être comprenez - ​ vous ses proches .
	Reference:  Ntango mosusu okoki kozala na likanisi ndenge moko na bato ya libota na ye .
 	Hypothesis: Mbala mosusu oyebi bandeko na ye .
```

Example 2
```ln
	Source:     L’endurance face à de telles épreuves est particulièrement précieuse pour Jéhovah .
	Reference:  Koyika mpiko liboso ya komekama wana ezali na motuya mingi na miso ya Yehova .
	Hypothesis: Koyika mpiko na mikakatano ya ndenge wana ezali na ntina mingi mpo na Yehova .
```

Example 3
```ln
	Source:     Résumé : Éliya apporte la preuve que Jéhovah est supérieur à Baal .
	Reference:  Na mokuse : Eliya amonisi ete Yehova aleki Baala .
	Hypothesis: Na mokuse : Eliya amonisaki ete Yehova aleki Baala .
```

Example 4
```ln
	Source:     Dans l’ensemble , David s’est montré un homme de foi .
	Reference:  Mokonzi Davidi azalaki moto ya sembo , mpe Yehova azalaki kolinga ye mingi .
	Hypothesis: Davidi azalaki moto ya kondima .
```

Example 5
```ln
	Source:     J’ai donc décidé de me confesser et , pour pénitence suprême , de mettre fin à mes jours .
	Reference:  Lokola namonaki ete nakokoka te kofuta mbongo yango , nakómaki kaka mawamawa .
	Hypothesis: Yango wana , nazwaki ekateli ya koyambola ngai , mpe mpo na nini nasengeli kotika mikolo na ngai .
```

# Results
	- BLEU dev :   26.41
	- BLEU test :  39.81

### CODE REVIEW

File: fr-ln/jw300-baseline/baseline.ipynb [hunk 1]
```diff
@@ -0,0 +1,6194 @@
+{
```
Comment (jaderabbit) [inline fr-ln/jw300-baseline/baseline.ipynb]: <p>what do you mean by "french swc" test set? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (jaderabbit) [inline fr-ln/jw300-baseline/baseline.ipynb]: <p>So, people were have issues with this on colab. Did you train locally?</p><p> </p><p>Maybe we can add comments to this affect (to this and to the starter_notebook):</p><p> </p><p><code>#If you're not running on colab, uncomment the lines below as they'll help your code run faster</code></p><p><code>&lt;your code here&gt;</code></p><p><code> </code></p><p><code>#If you are running on colab, the lines below will work - but they do take long for big datasets</code></p><p><code>&lt;put old code here&gt;</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

File: language_pairs.md [hunk 2]
```diff
@@ -30,3 +30,4 @@
 | English | Yoroba |  36.74   | [link](https://github.com/masakhane-io/masakhane/tree/master/en-yo/jw300-baseline) |
 | English | isiZulu (Autshumato) |  1.96   | [link](https://github.com/masakhane-io/masakhane/tree/master/en-zu/autshumato-baseline) |
 | English | isiXhosa (Autshumato) | 13.32 | [link](https://github.com/masakhane-io/masakhane/tree/master/en-xh/autshumato-baseline) |
+| French | Lingala | 39.81 | [link](https://github.com/masakhane-io/masakhane/tree/master/fr-ln/french-lingala-baseline) |
```
Comment (jaderabbit) [inline language_pairs.md]: yay! Thank you for updating this!

File: fr-ln/jw300-baseline/baseline.ipynb (hunk 1 above)
Comment (espoirMur) [reply fr-ln/jw300-baseline/baseline.ipynb]: <p>Okay...</p><p>That would be nice...</p><p>I used the code to train on GCP but not locally on my personal laptop.</p><p>I would be nice if you tell me which specific issues people are having on Colab.</p><p>I will definitely add those line both on Colab and this notebook...</p>

Comment (espoirMur) [reply fr-ln/jw300-baseline/baseline.ipynb]: <p>It's should be french Lingala, I copy and paste from another notebook..</p><p>I will definitely update...</p>

File: fr-ln/jw300-baseline/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Author: Murhabazi B. Espoir
+
+## Data
+
+	- The JW300 French - Lingala.
+
+## Model
+
+	- Default Masakhane Transformer translation model.
+	- The model can be found [here](https://drive.google.com/open?id=tobeUpdated)
```
Comment (juliakreutzer) [inline fr-ln/jw300-baseline/README.md]: Do you have a link yet?

File: fr-ln/jw300-baseline/baseline.ipynb (hunk 1 above)
Comment (espoirMur) [reply fr-ln/jw300-baseline/baseline.ipynb:1]: Fixed...🤦🏾‍♂️

Comment (espoirMur) [reply fr-ln/jw300-baseline/baseline.ipynb:1]: Are people still having issues with this in the starter notebooks? 🤔
@jaderabbit ?

File: fr-ln/jw300-baseline/README.md (hunk 3 above)
Comment (juliakreutzer) [reply fr-ln/jw300-baseline/README.md]: Thanks for adding it!

**Comment by juliakreutzer** [comment]
Hi @espoirMur, you might have noticed the change in directories: now all the baseline models are in 'benchmarks'. Your edits added all previous models back to the original structure. Can you please remove these other benchmarks from your PR and only keep your new benchmark? It might also be easier to open a new PR starting from an updated master. 
Please also make sure to link to or upload trg_vocab.txt, src_vocab.txt, config.yaml, and the bpe model. Thank you! :smile:

**Comment by espoirMur** [comment]
> Hi @espoirMur, you might have noticed the change in directories: now all the baseline models are in 'benchmarks'. Your edits added all previous models back to the original structure. Can you please remove these other benchmarks from your PR and only keep your new benchmark? It might also be easier to open a new PR starting from an updated master.
> Please also make sure to link to or upload trg_vocab.txt, src_vocab.txt, config.yaml, and the bpe model. Thank you! 😄
Thanks for pointing out that ...
I am done with that, it's now fixed I think!

**Comment by espoirMur** [comment]
> Thanks so much for the edits, @espoirMur!
> One more thing: please remove the .gitignore file.
> Otherwise it looks all good!

I have edited and move the .gitignore file to my local order to avoid pushing the large .ckpt file

**Comment by juliakreutzer** [comment]
@jaderabbit GitHub says you requested changes, are they now are all done? Please approve the PR if that is the case :) thanks!

**Pull request merged**